### DISCUSSION TITLE
Clarification - can we change "Delete" to "Delete Connection" in connection dropdown menu?

### DISCUSSION THREAD

**Discussion by troogyt** [dbeaver/dbeaver#33393] [Ideas]
In the screenshot below, does "**Delete**" mean "_Delete the database_", or does it mean "_Delete the connection to the database, but leave the database in tact_"?
![image](https://github.com/dbeaver/dbeaver/assets/41710820/778adeff-310a-4b52-9243-167781411e46)

If my guess is correct, and it means "_Delete only the connection to the database but do not delete the database_", then can we please change the menu to read as follows?
![image](https://github.com/dbeaver/dbeaver/assets/41710820/6760ab29-38c2-4e6c-8623-e84c94ccd909)

I am not that smart, so when I read this I get worried that I'm going to delete the actual database.
Making this minor change would remove this stress and make it much clearer (at least, for us dummies :))

**Comment by E1izabeth** [comment]
We created the issue based on your suggestion #33618

**Comment by E1izabeth** [comment]
This `Delete` button here means to remove connection from the list

**Reply by eriklien** [reply]
"Remove connection" would be even better! (I was a bit anxious yesterday when I was about to click Delete to remove a connection, so I had to double-check that this would not delete anything else - changing the text would help.)
